chef-universal-mingw32.gemspec

```diff
@@ -15,7 +15,7 @@ gemspec.add_dependency "win32-taskscheduler", "~> 2.0"
 gemspec.add_dependency "win32-certstore", "~> 0.6.15"
 gemspec.add_dependency "wmi-lite", "~> 1.0"
 gemspec.add_dependency "iso8601", ">= 0.12.1", "< 0.14" # validate 0.14 when it comes out
-gemspec.add_dependency "chef-powershell", "~> 1.0.12" # 0.5+ required for specifying user vs. system store
+gemspec.add_dependency "chef-powershell" # 0.5+ required for specifying user vs. system store
```

Comment:
Surprised this is working, given UCRT build. But if you're going for "18.1.0" I'd at least do "~> 18.1.0"... and the comment is irrelevant now.

Reply:
removed the comment